packages/components/ng-at-internet/src/index.js

```diff
@@ -14,7 +13,6 @@ const moduleName = 'ngAtInternet';
 
 angular
   .module(moduleName, [])
-  .constant('AT_INTERNET_CUSTOM_VARS', AT_INTERNET_CUSTOM_VARS)
```

Comment:
I don't think it is, because it's just moved, it isn't removed. 

It has been moved to the package `ovh-at-internet` which is used by this package, in the `provider.js`